### PR TITLE
Feature/activity repo

### PR DESCRIPTION
**Pull request by peytonjo**
#### What does this PR do?
- builds out dom update files, gets accessibility to 97%, adds mixins, and includes to scss.
#### Where should the reviewer start?
- src/activityRepository.js, src/css/_buttons.scss, src/css/_headers.scss
#### How should this be manually tested?
- run "npm start"
#### Any background context you want to provide?
- N/A

### CODE REVIEW

**Review by demaceo — APPROVED** [review]
phenomenal! aria labels & scss are 👌 👌

File: src/css/_headers.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  text-align: center;
+  height: 9vw;
+}
+
+button:active {
+  border: none;
+  outline: none;
+}
+
+#profile-button {
```
Comment (JWoltjen) [inline src/css/_headers.scss:11]: I'm still not sure why we can't get the SCSS to work when we try to break it down into smaller subfiles. Maybe something to circle back on if we have the time.

File: src/index.html [hunk 2]
```diff
@@ -22,210 +22,210 @@ <h5 id='dropdown-name'></h5>
     <main>
```
Comment (JWoltjen) [inline src/index.html:22]: Awesome use of ARIA tags to help us get to 97%

File: src/scripts.js [hunk 3]
```diff
@@ -19,40 +19,97 @@ function getData() {
     let todaysDate = moment().format('YYYY/MM/DD');
     let userRepository = new UserRepository(data, todaysDate);
     currentUser = userRepository.users[Math.floor(Math.random() * 50)]
-    domeUpdates.defineData(currentUser, todaysDate, userRepository);
+    domUpdates.defineData(currentUser, todaysDate, userRepository);
```
Comment (JWoltjen) [inline src/scripts.js:22]: Nice catch.

**Review by JWoltjen — APPROVED** [review]
Great job with these changes!

**Pull request merged**